### PR TITLE
feat: trailing comma in function calls

### PR DESCRIPTION
**Pull request by evilebottnawi**
/cc @czosel 

+ some refactor, we need cover this cases, let's do it in other PRs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ Prettier for PHP supports the following options:
...
 | `singleQuote`   | `false`   | If set to `"true"`, strings that use double quotes but do not rely on the features they add, will be reformatted. Example: `"foo" -> 'foo'`, `"foo $bar" -> "foo $bar"`.                                                                                                                                                   |
-| `trailingCommaPHP` | `"none"`  | If set to `"all"`, trailing commas will be added wherever possible. <br> If set to `"php7.2"`, trailing commas will be added to multiline arrays, lists and uses. <br> If set to `"php5"`, trailing commas will be added to multiline arrays and lists. <br> if set to `"none"`, no trailing commas.                       |
+| `trailingCommaPHP` | `"none"`  | If set to `"all"`, trailing commas will be added wherever possible. <br> If set to `"php7.3"`, trailing commas will be added to function calls. <br> <br> If set to `"php7.2"`, trailing commas will be added to multiline arrays, lists and uses. <br> If set to `"php5"`, trailing commas will be added to multiline arrays and lists. <br> if set to `"none"`, no trailing commas.                       |
```
Comment (czosel) [inline README.md]:
```suggestion
| `trailingCommaPHP` | `"none"`  | If set to `"all"`, trailing commas will be added wherever possible. <br> If set to `"php7.3"`, trailing commas will be added to multiline arrays, lists, uses and function calls (same as `"all"`). <br> If set to `"php7.2"`, trailing commas will be added to multiline arrays, lists and uses. <br> If set to `"php5"`, trailing commas will be added to multiline arrays and lists. <br> if set to `"none"`, no trailing commas.                       |
```

File: src/options.js [hunk 2]
```diff
@@ -20,6 +20,10 @@ module.exports = {
         description:
           "Trailing commas where valid in PHP 7.2 (arrays, lists and uses)."
       },
+      {
+        value: "php7.3",
+        description: "Trailing commas where valid in PHP 7.3 (function calls)."
```
Comment (czosel) [inline src/options.js]:
```suggestion
        description: "Trailing commas where valid in PHP 7.3 (arrays, lists, uses and function calls)."
```

File: README.md (hunk 1 above)
Comment (czosel) [reply README.md]: (that's how it actually works, right?)

**Comment by czosel** [comment]
Looks good for the rest! :+1:

File: README.md (hunk 1 above)
Comment (evilebottnawi) [reply README.md]: Yes :+1:

**Pull request merged**